platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheGracefulDegradationIntegrationTest.groovy

```diff
@@ -118,9 +199,9 @@ class ConfigurationCacheGracefulDegradationIntegrationTest extends AbstractConfi
 
         where:
         expectedProblems                                                                                                | _
-        ["Build file 'build.gradle': line 10: invocation of 'Task.project' at execution time is unsupported."]          | _
-        ["Build file 'build.gradle': line 10: invocation of 'Task.project' at execution time is unsupported.",
-         "Build file 'build.gradle': line 14: invocation of 'Task.taskDependencies' at execution time is unsupported."] | _
+        [INVOCATION_OF_TASK_PROJECT_AT_EXECUTION_TIME]                                                         | _
+        [INVOCATION_OF_TASK_PROJECT_AT_EXECUTION_TIME,
+         "invocation of Task.taskDependencies at execution time is unsupported with the configuration cache."] | _
```

Comment:
Refactoring the string as constant broke something here, looking into it

Reply:
Ah, that is for task dependencies. Will refactor ir as well (even though there is only one call site) for consistency